awx/main/tests/functional/test_inventory_source_injectors.py

```diff
@@ -192,10 +193,15 @@ def create_reference_data(source_dir, env, content):
 
 
 @pytest.mark.django_db
-@pytest.mark.parametrize('this_kind', CLOUD_PROVIDERS)
-def test_inventory_update_injected_content(this_kind, inventory, fake_credential_factory, mock_me):
+def test_inventory_update_injected_content(inventory, fake_credential_factory, self):
     ExecutionEnvironment.objects.create(name='Control Plane EE', managed=True)
     ExecutionEnvironment.objects.create(name='Default Job EE', managed=False)
+    cloud_providers = discover_available_cloud_provider_plugin_names()
```

Comment:
This is good, but could optionally move into a dedicated fixture.

Reply:
I will leave this open to loop back to once I resolve the other failures. Thank you!